docs/examples/pydata.md

```diff
@@ -91,10 +91,21 @@ data = xr.DataArray(
 data
 ```
 
+## ipyleaflet
+
+`ipyleaflet` is a **Jupyter**/**Leaflet** bridge enabling interactive maps in the Jupyter notebook environment. this demonstrate how you can integrate maps in your documentation.
+
+```{code-cell}
```

Comment:
The map is shown twice in the doc, maybe removing the last m will do the trick.

Reply:
Hmmm I tried a few different approaches and have no idea why that map is showing up twice now. Could we just add a comment or open an issue to track de-bugging it? I suspect this might be a bug upstream and not related to this theme or the code in the cell :-/